src/datasource/plugin.json

```
@@ -6,7 +6,7 @@
   "annotations": true,
   "backend": true,
   "alerting": true,
-  "executable": "../gpx_zabbix-plugin",
+  "executable": "gpx_zabbix-plugin",
```

It might be a nice idea to also update this to 
```suggestion
  "executable": "gpx_zabbix-datasource",
```